ros-vm-windows/InstallVstsAgent.ps1

```diff
@@ -0,0 +1,110 @@
+# Downloads the Visual Studio Team Services Build Agent and installs on the new machine
+# and registers with the Visual Studio Team Services account and build agent pool
+
+# Enable -Verbose option
+[CmdletBinding()]
+Param
+(
+	[Parameter(Mandatory=$true)]
+	[string]$VSTSAccount,
+
+	[Parameter(Mandatory=$true)]
+	[string]$PersonalAccessToken,
+
+	[Parameter(Mandatory=$true)]
+	[string]$AgentName,
+
+	[Parameter(Mandatory=$true)]
+	[string]$PoolName,
+
+	[Parameter(Mandatory=$true)]
+	[int]$AgentCount,
+
+	[Parameter(Mandatory=$true)]
+	[string]$AdminUser,
+
+	[boolean]$prerelease=$false
+)
+
+Write-Verbose "Entering InstallVSOAgent.ps1" -verbose
+
+$currentLocation = Split-Path -parent $MyInvocation.MyCommand.Definition
+Write-Verbose "Current folder: $currentLocation" -verbose
+
+#Create a temporary directory where to download from VSTS the agent package (vsts-agent.zip) and then launch the configuration.
+$agentTempFolderName = Join-Path $env:temp ([System.IO.Path]::GetRandomFileName())
+New-Item -ItemType Directory -Force -Path $agentTempFolderName
+Write-Verbose "Temporary Agent download folder: $agentTempFolderName" -verbose
+
+$serverUrl = "https://dev.azure.com/$VSTSAccount"

```

Comment:
will this work on every instance, even if this format has not been enabled?

Reply:
@bmoore-msft 

> will this work on every instance, even if this format has not been enabled?

What does that mean? Honestly, the script is shamelessly copied from https://github.com/Azure/azure-quickstart-templates/blob/master/201-vm-vsts-agent/scripts/InstallVstsAgent.ps1. I only tested it with my instances and I might not have full test coverage on everything.

Reply:
Azure DevOps allows accessing by

accountname.visualstudio.com
or
dev.azure.com/accountname

But I just tried an old one I have and both seem to work fine...